examples/docs_snippets/docs_snippets/concepts/assets/asset_checks/factory.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any, Mapping, Sequence
+
+from dagster import AssetCheckResult, Definitions, asset, asset_check
+
+
+@asset
+def orders():
+    ...
+
+
+@asset
+def items():
+    ...
+
+
+def make_checks(check_blobs: Sequence[Mapping[str, str]]):
```

Comment:
A return type would be helpful for understanding what this code is doing

Reply:
👍 I hadn't added AssetChecksDefinition to the top level imports yet, will fix